source/ed/connectors/osm_tags_reader.cpp

```diff
@@ -153,7 +153,7 @@ std::bitset<8> parse_way_tags(const std::map<std::string, std::string>& tags) {
                 bike_direct = bike_lane;
         } else if (key == "bicycle") {
             if (val == "yes" || val == "permissive" || val == "destination" || val == "designated" || val == "private"
-                || val == "true" || val == "allowed" || val == "official" || val == "destination")
+                || val == "true" || val == "allowed" || val == "official")
```

Comment:
Are you sure you want to remove `destination` ? 
From https://wiki.openstreetmap.org/wiki/Bicycle#Bicycle_Restrictions, I understand that the access by bike is only allowed if the street is you destination. 

I guess this is now a problem is you want to cycle to your place, and you leave in a street that have that `destination` restriction.  :thinking:

Reply:
`val == "destination"` was there twice in the code, I simply removed one :)